bgpd/bgp_vty.c

```diff
@@ -10958,6 +10958,8 @@ static int bgp_show_summary(struct vty *vty, struct bgp *bgp, int afi, int safi,
 
 					vty_out(vty, " %8u", 0);
 				}
+				if (peer->desc)
+					vty_out(vty, " %s", peer->desc);
```

Comment:
We also need to print something N/A or so if the description does not exist maybe?

Reply:
ok

Reply:
Did you remove n/a from vtysh output? IMO it should be added, because what happens when you add something more into this output after Description in the future?

Reply:
Also, I think truncating to 16 chars or so, would be good as well, what do you think?

Reply:
for truncate, you mean on the vty output ? I am not sure about this, coz it is the last column, so we dont have any restriction.

Reply:
But I think will be added more in the future, like PfxSnt and Desc were added in the last few months?